packages/api-sdk/src/medical/client/metriport.ts

```diff
@@ -452,6 +454,26 @@ export class MetriportMedicalApi {
     return documentQuerySchema.parse(resp.data);
   }
 
+  /**
+   * Start a bulk document download for a given patient, with the payload returned to the webhook.
```

Comment:
"...with the payload returned to the webhook" > "...with the payload sent via webhook"

I'd also add a `@see` element:

```ts
@see {@link <link-to-the-docs>}
```